index.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
-exports.htmlEscape = string => string
+exports.htmlEscape = string => String(string)
 	.replace(/&/g, '&amp;')
 	.replace(/"/g, '&quot;')
 	.replace(/'/g, '&#39;')
 	.replace(/</g, '&lt;')
 	.replace(/>/g, '&gt;');
 
-exports.htmlUnescape = htmlString => htmlString
+exports.htmlUnescape = htmlString => String(htmlString)
```

Comment:
I don't think these methods should coerce to a string. I think it should happen in `htmlEscapeTag` and `htmlUnescapeTag`. This would also fit with the existing TS types.